unit_conversion/lat_long.py

```diff
@@ -267,14 +267,14 @@ def format(self, style):
     def format_html(self, style):
         """
         format_html(style)
-        
+
         Backward compatibility for Quixote rlink and Pylons inews.
         """
-        return self.format(style).replace(u"\xb0", u"&deg;").encode("ascii")
+        return self.format(style).replace(u"\xb0", u"&deg;")
 
```

Comment:
it may be important that this be ascii. though maybe ascii-compatible unicode string is OK.

I'm checking with @mikeorr -- he's the one that's used this (and wrote it)